vumi_http_proxy/blacklist_reader.py

```diff
@@ -0,0 +1,23 @@
+import yaml
+import sys
+from twisted.python import log
+
+
+class FileReader(object):
+    def __init__(self, blacklistfile):
```

Comment:
We can probably turn this class into just a function since it doesn't really need any state. So something like:
```python
def read_blacklist(blacklistfile):
      ...
      return blacklist
```